### PR TITLE
Add AssetAttributes on assets entity

### PR DESCRIPTION
**Pull request by AlexandrosKyriakakis**

### CODE REVIEW

File: alpaca/entities.go [hunk 1]
```diff
@@ -116,19 +116,28 @@ type Position struct {
...
+	Tradable                     bool             `json:"tradable"`
+	Marginable                   bool             `json:"marginable"`
+	MaintenanceMarginRequirement uint             `json:"maintenance_margin_requirement"`
+	Shortable                    bool             `json:"shortable"`
+	EasyToBorrow                 bool             `json:"easy_to_borrow"`
+	Fractionable                 bool             `json:"fractionable"`
+	Attributes                   []AssetAttribute `json:"attributes"`
 }
 
+type AssetAttribute string
```
Comment (gnvk) [inline alpaca/entities.go:134]: Any advantage of using a custom type here vs simple strings? Especially that this seems to be a highly flexible field.

Comment (AlexandrosKyriakakis) [reply alpaca/entities.go:134]: I used custom type just to be able to match the existing attributes `ptp_with_exception` `ptp_no_exception`.
I can update it with  `[]string` if you think it is better for this use case.

Comment (gnvk) [reply alpaca/entities.go:134]: You can easily match with strings. I'd just go with them.

**Review by gnvk — APPROVED** [review]
LGTM

**Pull request merged**